src/NServiceBus.Core/DelayedDelivery/TimeoutManager/TimeoutManager.cs

```diff
@@ -69,11 +55,95 @@ protected internal override void Setup(FeatureConfigurationContext context)
 
                 return new ExpiredTimeoutsPoller(b.Build<IQueryTimeouts>(), b.Build<IDispatchMessages>(), dispatcherAddress, circuitBreaker);
             }, DependencyLifecycle.SingleInstance);
+
+            context.RegisterStartupTask(b => new TimeoutPollerRunner(b.Build<ExpiredTimeoutsPoller>()));
+        }
+
+        static string SetupDispatcherSatellite(FeatureConfigurationContext context, RetryStorageCleaner storageCleaner)
+        {
+            var errorQueueAddress = ErrorQueueSettings.GetConfiguredErrorQueue(context.Settings);
```

Comment:
@johnsimons Good catch. I removed code duplication.

@timbussmann @andreasohlund Let's handle the 2 second / 2 minute with this reopened issue.